cmd/libsnap-confine-private/cgroup-freezer-support.c

```diff
@@ -19,51 +37,9 @@ static const char *freezer_cgroup_dir = "/sys/fs/cgroup/freezer";
 
 void sc_cgroup_freezer_join(const char *snap_name, pid_t pid)
 {
-	// Format the name of the cgroup hierarchy.
 	char buf[PATH_MAX] = { 0 };
```

Comment:
Renaming to `snap_security_tag` increases readability now I thinkl.

Reply:
This is not the security tag however. It is really the snap name alone.